app/models/manageiq/providers/ansible_tower/shared/inventory/parser/automation_manager.rb

```diff
@@ -48,6 +48,13 @@ def configuration_script_sources
       inventory_object = persister.configuration_script_sources.find_or_build(project.id.to_s)
       inventory_object.description = project.description
       inventory_object.name = project.name
+      inventory_object.authentication = persister.credentials.lazy_find(project.credential_id.to_s)
```

Comment:
cc @Ladas it would be nice if `lazy_find` would also work with non strings - maybe this is just a small tweak somehwere